peerconnection.go

```diff
@@ -1118,10 +1118,18 @@ func (pc *PeerConnection) SetRemoteDescription(desc SessionDescription) error {
 			case direction == RTPTransceiverDirectionRecvonly:
 				if t.Direction() == RTPTransceiverDirectionSendrecv {
 					t.setDirection(RTPTransceiverDirectionSendonly)
+				} else if t.Direction() == RTPTransceiverDirectionRecvonly {
+					t.setDirection(RTPTransceiverDirectionInactive)
 				}
 			case direction == RTPTransceiverDirectionSendrecv:
 				if t.Direction() == RTPTransceiverDirectionSendonly {
 					t.setDirection(RTPTransceiverDirectionSendrecv)
+				} else if t.Direction() == RTPTransceiverDirectionInactive {
```

Comment:
Transit `Inactive` transceiver to `RecvOnly` if the remote peer reuse the `inactive` transceiver to send track